gcloud/monitoring/query.py

```diff
@@ -0,0 +1,661 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Time series query for the `Google Monitoring API (V3)`_.
+
+.. _Google Monitoring API (V3):
+    https://cloud.google.com/monitoring/api/ref_v3/rest/v3/\
+    projects.timeSeries/list
+"""
+
+import copy
+import datetime
+import itertools
+
+from gcloud.monitoring._dataframe import _build_dataframe
+from gcloud.monitoring.timeseries import TimeSeries
+
+_NOW = datetime.datetime.utcnow  # To be replaced by tests.
```

Comment:
Tiny nit, can you rename as `_UTCNOW`. I ask because the existence of both `datetime.datetime.utcnow` and `datetime.datetime.now` make it potentially confusing.